src/Util/OpenGamePrompt.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc;
+using System.Diagnostics;
+
+namespace sodoff.Utils;
+
+public class OpenGamePrompt {
+    public static void prompt() {
+        string key;
+        Console.WriteLine("Do you want to open the game");
+        Console.WriteLine("yes or no");
+        key = Console.ReadLine();
+        if (key == "yes")
+        {
+            Console.WriteLine("Opening game");
+            Process.Start(Directory.GetParent(Environment.CurrentDirectory) + "/Client/" + "DOMain.exe");

```

Comment:
this should be configurable ... maybe instead of bool `Open Game` options use string with path to game exe (and use empty or null string as false)

Reply:
i added new option called Clientpath
it checks if path is ending with exe
if ClientPath option is empty it shows a message
Invalid or empty Configuration value, Check your configuration file appsettings.json ClientPath is incorrect